### DISCUSSION TITLE
Use with Zigbee devices ?

### DISCUSSION THREAD

**Discussion by githule** [Beormund/Tasmota32-Multi-Zone-Heating-Controller#18] [Announcements]
Hi,

I was looking at your code.
Thank you for this it's a good introduction for me to understand berry script and tasmota.

I am using zigbee devices (pairing is working, can talk, tasmota is installed on top of Son Off bridge). 

Do you think I can adapt your code to handle zigbee relays and temperature sensors ? Do you see limitations ?

I think I will need something to make some kind of overlay to handle matching between zone devices and respective zigbee components.

@Beormund  Would you be interested to get the code if I success that dev or do you consider this scenario out of the scope of this project ?

If you have some advice it's welcome.
I will work on it on next weeks. 

Thank you,

**Comment by Beormund** [comment]
Have migrated the issue to discussions. I'm not familiar with Zigbee on Tasmota. But I look forward to your findings and if you have anything to enhance the heating controller am always happy to consider pull requests.
